modules/bridge/src/main/ts/ephox/bridge/components/content/Autocompleter.ts

```diff
@@ -30,7 +30,9 @@ export interface AutocompleterItem {
 
 export interface AutocompleterSpec {
   type?: 'autocompleter';
-  ch: string;
+  // TODO: TINY-8929: Remove 'trigger' fallback to 'ch'
+  ch?: string;
+  trigger: string;
```

Comment:
If I'm reading this right it will be a breaking change, at least for TypeScript customers, requiring third party implementations to add `trigger`?

I don't know if there's a nice way to make this migration. Setting them both to optional will require extra validation to make sure at least one of them is supplied.

Reply:
I made it optional in autocompleter spec, but mandatory in autocompleter schema, so if none of this is applied there will be a runtime error: `Could not find valid *required* value for "trigger"`

Reply:
That's the way to do it 👍 

I'm not totally happy with the error message but that's just what we get out of boulder.